tests/end2end/features/utilcmds.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Miscellaneous utility commands exposed to the user.
+
+    Background:
+        Given I open data/scroll/simple.html
+        And I run :tab-only
+
+    ## :later
+
+    Scenario: :later before
+        When I run :later 500 scroll down
+        Then the page should not be scrolled
```

Comment:
I guess I could trick it into waiting 500ms somehow. Ordering dependencies between tests are annoying as hell.